.github/workflows/documentation_examples_check.yml

```diff
@@ -1,5 +1,14 @@
 name: documentation_examples_check
-on: push
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 1 * * *'
```

Comment:
Lets change this to [every two hours](https://cron.help/#0_*/2_*_*_*)


```
0 */2 * * *
```

Reply:
Thanks